db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -86,82 +95,998 @@ export function htmlToSimpleTextBlock(html: string): EnrichedBlockSimpleText {
+            }
+        }
+    throw new Error(
+        `Tried parsing a WP component but never found a matching end tag for ${componentDetails.tagName}`
+    )
+}
+
+/** Handles finishing a partially parsed WpComponent when we get to the closing tag. When
+    a tag contains all the information needed to turn it into an OwidEnrichedArticleBlock then
+    we create that - otherwise we keep the WpComponent around with the children content filled in */
+function finishWpComponent(
+    details: WpComponent,
+    content: BlockParseResult<ArchieBlockOrWpComponent>
+): BlockParseResult<ArchieBlockOrWpComponent> {
+    return match(details.tagName)
+        .with("column", (): BlockParseResult<ArchieBlockOrWpComponent> => {
+            return {
+                content: [
+                    {
+                        ...details,
+                        childrenResults: content.content,
+                    },
+                ],
+                errors: content.errors,
+            }
+        })
+        .with("columns", () => {
+            const errors = content.errors
+            if (content.content.length !== 2) {
+                errors.push({
+                    name: "columns block expects 2 children",
+                    details: `Got ${content.content.length} children instead`,
+                })
+                return { ...content, errors }
+            }
+            const firstChild = tryGetAsWpComponentOfType(
+                content.content[0],
+                "column"
+            )
+            if (firstChild === undefined) {
+                errors.push({
+                    name: "columns block expects children to be column components",
+                    details: `Got ${firstChild} child instead`,
+                })
+                return { ...content, errors }
+            }
+            const secondChild = tryGetAsWpComponentOfType(
+                content.content[1],
+                "column"
+            )
+            if (secondChild === undefined) {
+                errors.push({
+                    name: "columns block expects children to be column components",
+                    details: `Got ${secondChild} child instead`,
+                })
+                return { ...content, errors }
+            }
+            return {
+                errors,
+                content: [
+                    {
+                        type: "sticky-right",
+                        left: convertAllWpComponentsToArchieMLBlocks(
+                            firstChild.childrenResults
+                        ),
+                        right: convertAllWpComponentsToArchieMLBlocks(
+                            secondChild.childrenResults
+                        ),
+                        parseErrors: [],
+                    } as EnrichedBlockStickyRightContainer,
+                ],
+            }
+        })
+        .with("paragraph", () => {
+            return content
+        })
```

Comment:
Since `wp:paragraph` are ignored in `wpComponentTagsToIgnore`, it seems to me that this will never be reached

Reply:
True, this is a leftover from an earlier iteration